packages/fresh_dio/test/fresh_test.dart

```diff
@@ -432,7 +432,7 @@ void main() {
         when(() => request.followRedirects).thenReturn(false);
         when(() => request.maxRedirects).thenReturn(0);
         when(() => request.listFormat).thenReturn(ListFormat.csv);
-        final response = MockResponse();
+        final response = MockResponse<dynamic>();
```

Comment:
Thanks so much for the PR! I can take a look at the analyzer issues because it looks like they're unrelated to this change.